Makefile

```diff
@@ -3,11 +3,30 @@ ODF_METADATA_CRATE_DIR=./src/odf/metadata
 LICENSE_HEADER=docs/license_header.txt
 TEST_LOG_PARAMS=RUST_LOG_SPAN_EVENTS=new,close RUST_LOG=debug
 
-POSTGRES_CRATES := ./src/infra/accounts/postgres ./src/infra/auth-rebac/postgres ./src/infra/datasets/postgres ./src/infra/task-system/postgres ./src/infra/flow-system/postgres ./src/infra/messaging-outbox/postgres ./src/e2e/app/cli/postgres
-
-MYSQL_CRATES := ./src/infra/accounts/mysql ./src/e2e/app/cli/mysql
-
-SQLITE_CRATES := ./src/infra/accounts/sqlite ./src/infra/auth-rebac/sqlite ./src/infra/datasets/sqlite ./src/infra/task-system/sqlite ./src/infra/flow-system/sqlite ./src/infra/messaging-outbox/sqlite ./src/e2e/app/cli/sqlite
+POSTGRES_CRATES := \
+	./src/e2e/app/cli/postgres \
+	./src/infra/accounts/postgres \
+	./src/infra/accounts/postgres \
+	./src/infra/auth-rebac/postgres \
+	./src/infra/datasets/postgres \
+	./src/infra/flow-system/postgres \
+	./src/infra/messaging-outbox/postgres \
+	./src/infra/task-system/postgres \
+	./src/infra/task-system/postgres \
+
+MYSQL_CRATES := \
+	./src/e2e/app/cli/mysql \
+	./src/infra/accounts/mysql \
+
+SQLITE_CRATES := \
+	./src/e2e/app/cli/sqlite \
+	./src/infra/accounts/sqlite \
+	./src/infra/accounts/sqlite \
```

Comment:
2 accounts

Reply:
Oh, interesting -- in linear form means we had duplicates that were spotted after sorting them out

Fixed